dot/network/block_announce_test.go

```diff
@@ -35,19 +35,22 @@ func TestBlockAnnounce_Encode(t *testing.T) {
 		Number:         big.NewInt(77),
 		StateRoot:      common.Hash{2},
 		ExtrinsicsRoot: common.Hash{3},
-		Digest:         types.Digest{},
+		Digest:         types.Digest(nil),
```

Comment:
@noot Will these changes from {} to nil on the blockAnnounceMessage.Digest also need to be reverted?